app/views/brexit_checker_mailer/change_notification.text.erb

```diff
@@ -1,4 +1,4 @@
-<%= t("brexit_checker_mailer.change_notification.#{@notification.type}") %>
+## <%= t("brexit_checker_mailer.change_notification.#{@notification.type}") %>
```

Comment:
This is markdown in an erb right?
The notify api will take this and turn it into an `h2`...
... and it's not an ERB comment?

(just thought i'd check, confusing with the file names!)

Reply:
Yes, that's right. Apparently an ERB comment is like `<%#`, although I've never used them myself.

(I need to force-generate an email to double-check the new formatting.)